apps/src/templates/projects/ProjectsList.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import {Table} from 'reactabular';
+import color from "../../util/color";
+
+const styles = {
+  cell: {
+    border: '1px solid gray',
+    padding: 10,
+    fontSize: 14,
+  },
+  nameCell: {
+    border: '1px solid gray',
+    padding: 10,
+    fontSize: 14,
+    fontFamily: '"Gotham 5r", sans-serif',
+  },
+  headerCell: {
+    border: '1px solid gray',
+    padding: 10,
+    backgroundColor: color.teal,
+  },
+};
+
+/**
+ * Map from project type to friendly name.
+ * @type {Object}
+ */
+const PROJECT_TYPE_MAP = {
+  applab: 'App Lab',
+  gamelab: 'Game Lab',
+};
+
+function typeFormatter(type) {
+  return PROJECT_TYPE_MAP[type];
+}
+
+/**
+ * Takes a date formatted as YYYY-MM-DD and returns it as MM/DD/YYYY.
+ * @param {string} date
+ * @returns {string}
+ */
+function dateFormatter(date) {
+  const result = date.match(/^(\d\d\d\d)-(\d\d)-(\d\d)$/);
+  if (!result) {
+    return date;
+  }
+  const [year, month, day] = result.slice(1, 4);
+  return `${month}/${day}/${year}`;
```

Comment:
It turns out the javascript `Date` object does everything we need -- it can parse the format stored in the database (see new stub data for examples), and it can output the mm?/dd?/yyyy date format we want in the US for now. Based on what I see in storybook, it appears to handle time zones correctly:
<img width="494" alt="screen shot 2017-01-12 at 4 53 16 pm" src="https://cloud.githubusercontent.com/assets/8001765/21914507/b2c7c2fa-d8e8-11e6-9033-18769a68fc32.png">